### PR TITLE
added autocomplete command

### PR DESCRIPTION
**Pull request by thecodeassassin**

### CODE REVIEW

File: commands/completion.go [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+	"github.com/symbiosis-cloud/cli/pkg/symcommand"
+	"github.com/symbiosis-cloud/symbiosis-go"
+)
+
+type CompletionCommand struct {
+	Client      *symbiosis.Client
+	CommandOpts *symcommand.CommandOpts
+}
+
+var CompletionCommands = []string{"node-types", "regions", "roles"}
```
Comment (Styren) [inline commands/completion.go:20]: This looks incorrect

Comment (thecodeassassin) [reply commands/completion.go]: > This looks incorrect

Indeed, that was left over from another command that I used as a template.

**Review by Styren — APPROVED** [review]
Just one idea, otherwise LGTM! Looks great 🥳

File: README.md [hunk 2]
```diff
@@ -56,3 +56,12 @@ Currently the CLI is in Beta. We only expose a limited number of CLI commands cu
...
 
+### Autocomplete
+
+You can run:
+
+```bash
+sym completion --help
+```
+
+To get instructions on how to setup autocomplete for your specific shell.
```
[the remark this answers is not in the document]
Comment (thecodeassassin) [reply README.md:67]: That would basically be copy pasting the autocomplete help text, which seemed kind of redundant to me which is why I added this. I can do that but I am not sure that there's much value in it.

**Review by Styren — APPROVED** [review]

**Pull request merged**